Update Barcode Scanner Compatibility

TerryWarwick opened the pull request:
Updated document structure to capture a single list of compatible devices and provide a consolidated support summary

TerryWarwick committed:
Update Barcode Scanner Compatibility

Updated document structure to capture a single list of compatible devices and provide a consolidated support summary

msftclas commented:
@TerryWarwick,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

muhsinking reviewed (commented):
Hey terry, thanks for the pull request--are the category titles in the Compatible Hardware table supposed to be italicized? If not, I can fix them, otherwise I'll merge the PR

TerryWarwick committed:
Compatible Devices Updates

Removed italicized titles in Category column, moved Honeywell Voyager 1602g from 1D to 2D and updated Honeywell N5600 to N5680 per feedback from Honeywell.

muhsinking reviewed (approved):
Great, thanks for the change!
